tests/structured_conf/data/attr_classes.py

```diff
@@ -299,10 +299,10 @@ class WithTypedDict:
     dict: Dict[str, int] = {"foo": 10, "bar": 20}
 
 
-@attr.s(auto_attribs=True)
-class ErrorDictIntKey:
-    # invalid dict key, must be str
-    dict: Dict[int, str] = {10: "foo", 20: "bar"}
+# @attr.s(auto_attribs=True)
+# class ErrorDictIntKey:
+#     # invalid dict key, must be str
+#     dict: Dict[int, str] = {10: "foo", 20: "bar"}
```

Comment:
Instead of commenting it out, replace it by another type that is not supported. I suggest something like `dict: Dict[object, str] = {object(): "foo", object(): "bar"}` because I don't expect that arbitrary objects will be supported any time soon.